src/crimson/os/seastore/lba_manager/btree/lba_btree_node.h

```diff
@@ -33,17 +33,17 @@ using LBANode = FixedKVNode<laddr_t>;
  */
 struct lba_map_val_t {
   extent_len_t len = 0;  ///< length of mapping
-  paddr_t paddr;         ///< physical addr of mapping
+  pladdr_t pladdr;         ///< physical addr of mapping
```

Comment:
///< physical addr of mapping :  should be physical addr or logic addr of mapping.